api/docker-compose.yaml

```diff
@@ -34,6 +34,24 @@ services:
     ports:
       - 8081:8081
       - 8001:8001
+  public-api:
+    depends_on:
+      - db
+    build:
+      context: ./src/dev/server
+    user: ${UID}
+    working_dir: /w/api
+    volumes:
+      - ..:/w:cached
+      - gradle-cache:/.gradle
```

Comment:
Ah. So we set `GRADLE_USER_HOME` in the Dockerfile which points to this. So if you used a different volume here (e.g., `gradle-cache-public-api`), you'd be able to run this alongside the `api` service at the cost of gradle re-downloading all of the jars for each.